esp/public/media/scripts/program/modules/flag-results-page.js

```diff
@@ -59,6 +59,27 @@ function hideAll () {
     $j(".flag-detail").hide();
 }
 
+function approveAll () {
+    var r = confirm("Are you sure you'd like to approve all of these classes?");
+    if (r) {
+        $j("#program_form .btn-approve").click();
```

Comment:
This seems likely to be fragile: if someone e.g. changes the name of the button they might not think to check here.  Instead, it would be much better to loop through each of the classes and call `approve` above.  You can either stick a list of IDs in the template somewhere, or pull them from the HTML if necessary.